OrchardCore.Commerce/Migrations/OrderMigrations.cs

```diff
@@ -1,32 +1,28 @@
-using OrchardCore.ContentManagement.Metadata.Settings;
-using OrchardCore.ContentManagement.Metadata;
-using OrchardCore.Data.Migration;
 using OrchardCore.Commerce.Fields;
 using OrchardCore.Commerce.Settings;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+
+namespace OrchardCore.Commerce.Migrations;
 
-namespace OrchardCore.Commerce.Migrations
+/// <summary>
+/// Adds the order part to the list of available parts and the address field to the list of available fields.
+/// </summary>
+public class OrderMigrations : DataMigration
 {
-    /// <summary>
-    /// Adds the order part to the list of available parts and the address field to the list of available fields.
-    /// </summary>
-    public class OrderMigrations : DataMigration
-    {
-        IContentDefinitionManager _contentDefinitionManager;
+    private readonly IContentDefinitionManager _contentDefinitionManager;
 
-        public OrderMigrations(IContentDefinitionManager contentDefinitionManager)
-        {
-            _contentDefinitionManager = contentDefinitionManager;
-        }
+    public OrderMigrations(IContentDefinitionManager contentDefinitionManager) => _contentDefinitionManager = contentDefinitionManager;
```

Comment:
I have went ahead and broke down every line longer than 130 columns.
(wow I'm getting a lot of mileage out of global Regex search today!)

However I don't think you needed to concentrate on this at all. I mean we have a 150 column hard cap for a reason, right? While lines between 120 and 150 columns are discouraged, if we thought they are the end of the world we should've set the code analyzer's hard cap to 120.

Reply:
I interpret it as follows: break the code to 120 characters, unless it really hurts formatting or readability. Neither is the case here, so go with 120.

Examples of where I didn't mention it include mostly method signatures that would otherwise have to be broken into several lines, or method invocations.